lightning/lightning.go

```diff
@@ -185,6 +186,19 @@ func (l *Lightning) RunOnce() error {
 	return l.run(cfg)
 }
 
+func (l *Lightning) RunEmbeddedOnce(ctx context.Context, taskCfg *config.Config, logger *zap.Logger, g glue.Glue) error {
```

Comment:
This method doesn't have much difference with `RunOnce`, we reuse the `RunOnce` directly with little change?

Reply:
https://github.com/pingcap/tidb-lightning/pull/456/commits/6e4da0a4257a061a0882b3de76b3cafb661f0682